tools/wptserve/wptserve/config.py

```diff
@@ -69,6 +69,19 @@ def logger(self):
     def as_dict(self):
         return json_types(self.__dict__)
 
+    # Environment variables are limited in size so we need to prune the most egregious contributors
+    # to size, the origin policy subdomains.
+    def as_dict_for_wd_env_variable(self):
+        dict = self.as_dict()
+        dict["subdomains"] = [x for x in dict["subdomains"] if not x.startswith("op")]
```

Comment:
Maybe it would be better to write this something like
```py
for key in [("subdomains",), ("domains", "alt"), ("domains", ""), ("all_domains", "alt"), ("all_domains", ""), ("domains_set",), ("all_domains_set",)]
    target = dict
    for part in key[:-1]:
        target = target[part]
    value = target[key[-1]]
    if isinstance(value, dict):
        target[key[-1]] = {k:v for (k,v) in iteritems(value) if not k.startswith("op")}
    else:
        target[key[-1]] = [x for x in value if not x.startswith("op")]
```

Reply:
This seems rather clever to me, but I'm happy to do so if it's more Pythonic!

Reply:
Not sure it's more pythonic, but it is arguably more maintainable since we aren't repeating all the logic for every key